### ISSUE TITLE
Set up another image generator that is trained on Rivers and Weezer

### ISSUE THREAD

**Issue by riverscuomo** [riverscuomo/cuomputer#23]

**Comment by ArthurEmidio** [comment]
I can give it a try on it one more time (and hopefully this time to also help out on something else in the repository).

Since the last time we did this last year, I see the most interesting models nowadays are:

1. [Stable Diffusion XL (SDXL)](https://stablediffusionxl.com/): seems to be the best open-source model right now which does well with faces. Replicate [supports](https://replicate.com/blog/fine-tune-sdxl) fine-tuning it since August 2023.
2. [DreamBooth](https://dreambooth.github.io/): model by Google, but Replicate [says](https://replicate.com/blog/dreambooth-api) that SDXL has been consistently giving better results compared to it.
3. [LoRA](https://huggingface.co/papers/2106.09685): it's a faster model which we wouldn't have the "warm-up" problem that we used to face with the previous model. The disadvantage is that it doesn't do very well on faces, but we could try. See: https://replicate.com/docs/guides/fine-tune-an-image-model

Do you feel that the time to generate the image was something that annoyed people? I believe quality should still be the top priority, but would be good to hear your thoughts.

**Comment by ArthurEmidio** [comment]
I ran a fine-tune of SXDL through Replicate using 9 recent images found in Google and I'll need to play with the parameters. It's performing worse than the old model.

Prompt: *A painting of Rivers Cuomo playing the acoustic guitar near the lake*

![out-3](https://github.com/riverscuomo/cuomputer/assets/392601/0d4d4dac-953f-40b0-976d-74cfe9e7a208)

Once we get satisfactory results with you, we can look into how we can introduce multiple people's faces in the same model without degrading the end result.

**Comment by riverscuomo** [comment]
Exactly: I think the main issue for me is that the model was too slow to start up. Ideally it would be as fast as Dall E, which I think is what I'm using now.

The image you posted here looks like an Improvement on DALL E.

**Comment by ArthurEmidio** [comment]
Fair, I'll try to use LoRA to verify how it helps on speed and how it affects output quality.

I'll post continuous updates in this thread in the next 2-3 days.

**Comment by hh2010** [comment]
![image](https://github.com/user-attachments/assets/51e6aa1b-7736-416e-b76b-f205cb9185e6)

example of a subpar image to be made better
